index.html

```diff
@@ -192,10 +192,10 @@
                     <p>Want to invite your friends to Sparsh? Spread the word!</p>
                     <ul class="list-inline banner-social-buttons">
                         <li>
-                            <a href="#" class="btn btn-default btn-lg"><i class="fa fa-facebook fa-fw"></i></a>
+                            <a href="https://www.facebook.com/Sparsh.NITsurat" class="btn btn-default btn-lg"><i class="fa fa-facebook fa-fw"></i></a>
                         </li>
                         <li>
-                            <a href="#" class="btn btn-default btn-lg"><i class="fa fa-twitter fa-fw"></i></a>
+                            <a href="https://www.twitter.com/Sparsh_SVNIT" class="btn btn-default btn-lg"><i class="fa fa-twitter fa-fw"></i></a>
```

Comment:
Supposed to be a sharing link so users can directly tweet to Twitter and not a simple link to the Twitter page.